bundles/org.openhab.binding.kvv/src/main/resources/ESH-INF/thing/station.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="kvv"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="kvvstation">
+		<label>KVV Station</label>
+		<description>Train station for KVV Binding.</description>
+
+		<config-description>
+			<parameter name="stationId" type="text">
+				<label>Station ID</label>
+				<description>ID of the train station.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="maxTrains" type="integer" required="true" min="0" max="127">
+				<label>Maximum Trains Listed</label>
+				<description>Maximum number of trains listed.</description>
+			</parameter>
+			<parameter name="updateInterval" type="integer" required="true" min="0">
```

Comment:
Is min=0 a good choice? From the usability perspective you might want to use seconds. You can also specify `unit="s"`

Reply:
I wanted to leave as many settings-granularity to the user. If it is okay for you, I would like to suggest to keep the granularity of milli-seconds. But I would be fine to use seconds.

Reply:
It's up to you if ms or sec.. But you should specify a reasonable min. value. If the user enters 0, `scheduleWithFixedDelay()` will throw an IllegalArgumentException and if the value is very low, the scheduler will clog the system.

Reply:
WDYT?

Reply:
I have set the unit to seconds and the minimal interval to 1s.